repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasEntityStoreV2.java

```diff
@@ -1917,6 +1917,7 @@ private EntityMutationResponse deleteVertices(Collection<AtlasVertex> deletionCa
                 }
                 entity.setDeleteHandler(handler);
                 entity.setStatus(Status.DELETED);
+                entity.setUpdatedBy(RequestContext.get().getUser());
```

Comment:
What will be the value for updateBy attribute in Entity audits? Make sure both are same

Reply:
Also, do we have information on what all subscribers are listening to ATLAS_ENTITIES?
Make sure we do not break any existing listeners listening to these type of notifications if any

Reply:
Hi @nikhilbonte21 
`deleteVertices` method just updates entity attributes which are later consumed by both audits and notification listeners.
In the above case, the  username who actually deleted the glossary will be used in `updatedBy` field.
Attached screenshot for your reference.
<img width="428" alt="Screenshot 2023-12-06 at 12 08 41 PM" src="https://github.com/atlanhq/atlas-metastore/assets/151714378/573728de-90fc-4e41-98fd-61030a2204c4">